utils/schemed-kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/schemedkafka/source/UniversalKafkaSourceFactory.scala

```diff
@@ -232,21 +246,22 @@ class UniversalKafkaSourceFactory(
       deserializationSchema,
       recordFormatter,
       kafkaContextInitializer,
-      prepareKafkaTestParametersInfo(valueSchemaUsedInRuntime, preparedTopic.original),
+      prepareKafkaTestParametersInfo(valueSchemaUsedInRuntime, preparedTopic.original, defaultValuesForTestParameters),
       modelDependencies.namingStrategy
     )
   }
 
   private def prepareKafkaTestParametersInfo(
       runtimeSchemaOpt: Option[RuntimeSchemaData[ParsedSchema]],
-      topic: TopicName.ForSource
+      topic: TopicName.ForSource,
+      defaultValuesForTestParameters: Map[ParameterName, Expression]
   )(
       implicit nodeId: NodeId
   ): KafkaTestParametersInfo = {
     Validated
       .fromOption(
         runtimeSchemaOpt,
-        NonEmptyList.one(CustomNodeError(nodeId.id, "Cannot generate test parameters: no runtime schema found", None))
+        NonEmptyList.one(CustomNodeError("Cannot generate test parameters: no runtime schema found", None))
```

Comment:
Why is the nodeId removed here? Is it because of how the FE error is formatted? We may need it when there are multiple sources to identify which one has problems.

Reply:
I used the apply method with nodeId as an implicit param (I think it's more typesafe)